doc/api/util.markdown

```diff
@@ -551,6 +551,8 @@ util.isUndefined(null)
 
 ## util.log(string)
 
+    Stability: 0 - Deprecated
```

Comment:
Recommending a specific third party package would be problematic, but saying something like, "Use a third party userland module instead" would be ok.